src/Footer.js

```diff
@@ -0,0 +1,14 @@
+// Footer.js
+import './App.css'
+
+function Footer() {
+    return (
+        <footer>
+            <p>API Powered by <a href="https://api.nasa.gov/" target="_blank" rel="noreferrer">NASA</a> 🚀 
+                <span className="text-break">Made by <a href="https://sylviaraposo.com" target="_blank" rel="noreferrer">Sylvia</a></span> 🚀
```

Comment:
just a small suggestion-- instead of styling the text break to be display block, you could include <br> within your p element or other text element to create a line-break